kuma/wiki/forms.py

```diff
@@ -342,20 +339,21 @@ def clean_tags(self):
                 doc_tag = (DocumentTag.objects.filter(name__exact=tag)
                                               .values_list('name', flat=True))
 
-                if doc_tag:
-                    if doc_tag[0] != tag and doc_tag[0].lower() == tag.lower():
-                        dupe_tags.append(doc_tag[0])
-
                 # Write a log we can grep to help find pre-existing duplicate
                 # document tags for cleanup.
                 if len(doc_tag) > 1:
                     log.warn('Found duplicate document tags: %s' % doc_tag)
 
-            if dupe_tags:
-                raise forms.ValidationError(
-                    TAG_DUPE % {'tag': get_text_list(dupe_tags)})
+                if doc_tag:
```

Comment:
Would `first()` trigger another query with a `LIMIT 1`? I'm trying to avoid 2 queries here. I could drop the logging and use `first()`, though. Either we haven't hit the case of more than one tag being found or the logging isn't set up at the right log level to see the logs.